NoComponentDrag.py

```diff
@@ -143,21 +135,21 @@ def is_parametric_mode():
         pass
     return False
 
+def clear_ui_item(item):
+    if item:
+        item.deleteMe()
+
```

Comment:
Fixed the name to Snake case (monkey brain kicked in when writing the function and I just did what was natural for me)

Name is also more appropriately named to "clear" instead of clean or delete since the item will usually be replaced by another it isnt exactly being deleted so much as it is being cleared before the refresh

Reply:
You are correct on the different behavior w r t delete. OK!